omniqueue/src/backends/redis/mod.rs

```diff
@@ -188,6 +235,8 @@ impl<R: RedisConnection> RedisBackendBuilder<R> {
             .await
             .map_err(QueueError::generic)?;
 
+        check_eviction_policy(redis.clone()).await;
```

Comment:
It might make sense to move this under `start_background_tasks` and/or put in a separate spawned thread, that way we don't have to call this in multiple places and we don't have to potentially block other startup processes.